k8s.io/configmap-nginx.yaml

```diff
@@ -122,6 +122,13 @@ data:
 
         rewrite ^/(.*)?$    https://github.com/kubernetes/kubernetes/tree/master/$1 redirect;
       }
+      server {
+        server_name cs.kubernetes.io cs.k8s.io;
+        listen 80;
+        listen 443 ssl;
+
+        rewrite ^/$       https://k8s-code.appspot.com redirect;
```

Comment:
would it be worth passing along any the rest of the request URI as an argument to the search?

e.g. translate https://cs.k8s.io/foo -> https://k8s-code.appspot.com/?q=foo ?

Reply:
Done! thanks @ixdy